src/pages/Search/components/Heatmap.styles.js

```diff
@@ -0,0 +1,77 @@
+import styled, { css } from 'styled-components';
+
+export const HeatmapContainer = styled.section`
+  width: 100%;
```

Comment:
You should rather set the width to a fixed pixel size than the padding. Setting a padding, especially a large on like this, will take unnecessary space on smaller devices.

Btw: To make this responsive I used `overflow-y: scroll`. You can try it out and make it work on smaller devices as well.

Reply:
I set the width to 1200px but on mobile devices can't see the whole table

Reply:
fixed need to add `margin : 0 auto `